### ISSUE TITLE
Cursor is hardly visible

### ISSUE THREAD

**Issue by gaganyadav80** [shiftybody/atom-one-dark-coal#1]
The blinking cursor is hardly visible. If possible can you fix that?

![SCR-20221213-nhs](https://user-images.githubusercontent.com/47142137/207305548-88369861-378a-4b4d-ba94-e54faea3d8cb.png)
![SCR-20221213-nhx](https://user-images.githubusercontent.com/47142137/207305572-59589e94-e132-4cdb-a2e6-e253c0dfa7f5.png)
![SCR-20221213-nhi](https://user-images.githubusercontent.com/47142137/207305626-4804df8b-2f07-4829-92d0-1c76c4c7cf61.png)

Loved you theme,
Thank you.

**Comment by shiftybody** [comment]
Hi ! thanks for acknowledging the issue. 
I have corrected the theme and modified the "editorCursor.foreground" property to make the cursor much more visible. 

![image](https://user-images.githubusercontent.com/34991503/208835171-efa11791-9c21-47ec-acf1-2c769a9856e0.png)

![image](https://user-images.githubusercontent.com/34991503/208835416-4b59d883-8dff-4517-a589-30f1a6b4d774.png)

I hope I didn't overdo it.  😅

Either way, if you think I've made it even more unbearable, create a pull request or share your color modifications.

I promise to take less time for the next one.
